internal/controller/pgupgrade/utils.go

```diff
@@ -49,8 +49,8 @@ func (r *PGUpgradeReconciler) setControllerReference(
 			Kind:               "PGUpgrade",
 			Name:               owner.GetName(),
 			UID:                owner.GetUID(),
-			BlockOwnerDeletion: pointer.BoolPtr(true),
-			Controller:         pointer.BoolPtr(true),
+			BlockOwnerDeletion: initialize.Pointer(true),
```

Comment:
🤔 would this complain... if it wasn't clear that we're returning a pointer to a bool?

Reply:
Yes. The types in the struct are... established. When I change `true` to `9`, `go build` and `go test` tell me:

```
internal/controller/pgupgrade/utils.go:52:24: cannot use initialize.Pointer(9) (value of type *int) as type *bool in struct literal
```